### ISSUE TITLE
[Bug]: Incompatible version linking Android to Linux

### ISSUE THREAD

**Issue by jurjendevries** [simplex-chat/simplex-chat#4129]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Platform

Linux, Android

### OS version

Android 14, Ubuntu 24.04

### App version

Mobile 5.6.1, Desktop 5.7.0.5

### Current Behavior

After scanning the QR code from my desktop with the app I am getting the error "Incompatible version. Desktop app version 5.7.0.5 is not compatible with this app." Both have the latest stable versions from Google Play and https://simplex.chat/downloads/#desktop-app

### Expected Behavior

Being able to link the devices

### Steps To Reproduce

1. Download SimpleX from Google play.
2. Download SimpleX from https://github.com/simplex-chat/simplex-chat/releases/latest/download/simplex-desktop-x86_64.AppImage on Ubuntu 24.04
3. Install on both platforms
4. Scan QR code from the Android app.
5. See the error

**Comment by slimygreenfingers** [comment]
There needs to be much more co-ordination between desktop and mobile releases--neither version 6.3 or later is compatible with iOS versions on apple store? 

This was so annoying I decided to use something else :D

**Comment by ole-tange** [comment]
> There needs to be much more co-ordination between desktop and mobile releases--neither version 6.3 or later is compatible with iOS versions on apple store?

Agreed. In general try to be backwards compatible - at least a year back.

It is OK that the old version will not support the new features.

The earliest version on F-droid is 6.4 and even that is incompatible with Ubuntu Snap 6.3.6.0. So I do not even see a way I can have a setup where both are working.

As a developer I perfectly understand wanting to change the whole architecture in an incompatible way from version to version. But from a user's perspective this wont work: They will simply conclude that the program does not work (which is true from their perspective) and refuse to use it - which in turn will limit the adoption.

> This was so annoying I decided to use something else :D

I was on-boarding, so I am postponing that.

I am a bit sad, because from the description SimpleX seems better than Signal and I was looking forwards to sharing this with journalists.
